src/components/alerts-config/alerts-config.modal.html

```diff
@@ -9,12 +9,36 @@
 
     <ul class="list-group">
 
-      <li class="list-group-item" ng-repeat="alert in alerts" ng-class="{active: alert.active}" ng-click="toggleAlert(alert)">
-        <i class="fa" ng-class="alert.active ? 'fa-toggle-on' : 'fa-toggle-off'" />
-        {{alert.label}}
+      <li class="list-group-item" ng-repeat="alert in alerts" ng-class="{active: alert.active}">
+        <div ng-click="toggleAlert(alert)">
+          <i class="fa" ng-class="alert.active ? 'fa-toggle-on' : 'fa-toggle-off'" />
+          {{alert.label}}
+        </div>
+
+        <ul class="list-group recipients-list" ng-show="showRecipientList(alert)">
+
+          <li class="list-group-item" ng-repeat="member in members" ng-show="member.active" ng-click="toggleRecipient(member)">
+            {{member.email}}
+            <button type="button" class="close" aria-label="Close">
+              <span aria-hidden="true">&times;</span>
+            </button>
```

Comment:
Because the ng-click directive is on the <li> element it is triggered when the button is clicked. The button is really there more for UX purposes as it makes it more clear which members are receiving the alert as well as how to remove them.